inst/examples/keras/cloudml.yml

```diff
@@ -0,0 +1,10 @@
+gcloud:
+  project         : "rstudio-cloudml"
+  account         : "javier@rstudio.com"
+  region          : "us-east1"
+  runtime-version : "1.2"
```

Comment:
See previous comment, I think this might belong in the `cloudml` key?

Reply:
makes sense, made generic.

Reply:
I mean that the `runtime-version` key might belong in the `cloudml` rather the `gcloud` config section